dev-qt/qt-creator/qt-creator-9999.ebuild

```diff
@@ -115,15 +114,8 @@ done
 unset x
 
 PATCHES=(
-	"${FILESDIR}"/${PN}-4.12.0-dylib-fix.patch
-	"${FILESDIR}"/${PN}-4.12.0-libclangformat-ide.patch
 )
 
-llvm_check_deps() {
-	has_version -d "sys-devel/clang:${LLVM_SLOT}" && \
```

Comment:
why did you delete the check on sys-devel/clang?

Reply:
I don't really mind keeping the sys-devel/clang check, but I didn't see any purpose of the check when we removed dev-libs/libclangformat. In other words, I failed to see why Qt Creator needs this check when it only covers clang. I thought llvm and clang would be kept in sync without Qt Creator forcing it, but I might very well be wrong. Sorry.

Reply:
Just let me know if you want the PR updated with the `llvm_check_deps()` function preserved (without the libs/libclangformat check).